hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.tsx

```diff
@@ -119,9 +122,9 @@ export const CartCard = ({ hardware_id, quantity, error }: CartCardProps) => {
                     size="small"
                     className={styles.CartClose}
                     onClick={() => {
-                        handleRemove(hardware.id);
+                        handleRemove(hardware_id);
                     }}
-                    role="remove"
+                    data-testid={hardware_id.toString()}
```

Comment:
This is ok but usually with test ids we name them something a bit more descriptive e.g. `cart-item-<id>`